step.yml

```diff
@@ -0,0 +1,87 @@
+---
+title: Get OIDC Identity Token
+summary: The Step fetches an OIDC identity token.
+description: |-
+  The "Prepare App Store Release" Step allows you to streamline the process of preparing a new release for your iOS app in the Release Management. This Step leverages the Bitrise Public API to facilitate the creation and configuration of an App Store release in the Release Management.
+
+  By utilizing this Step, you can automate the initial stages of the release process and ensure a consistent and efficient deployment experience. Instead of manually navigating through the Release Management interface to create a release, the Step empowers you to initiate the release setup programmatically, saving valuable time and effort.
+
+  It's important to note that this Step doesn't create a release directly in the App Store Connect. Instead, it streamlines the process by generating a release in the [Release Management](https://devcenter.bitrise.io/en/release-management.html).
+website: https://github.com/bitrise-steplib/bitrise-step-get-identity-token
+source_code_url: https://github.com/bitrise-steplib/bitrise-step-get-identity-token
+support_url: https://github.com/bitrise-steplib/bitrise-step-get-identity-token/issues
+
+type_tags:
+- utility
+
+project_type_tags:
```

Comment:
NIT: pls add `kotlin-multiplatform`

Reply:
Ohh if this step is not specific to any project types, we can simply omit the `project_type_tags` property (similarly to the [deploy-to-bitrise-io](https://github.com/bitrise-steplib/steps-deploy-to-bitrise-io/blob/master/step.yml#L74) step).